lib/trade_tariff_frontend.rb

```diff
@@ -6,7 +6,7 @@ module TradeTariffFrontend
   module_function
 
   def production?
-    environment == 'production'
+    environment == 'production' && ENV['PREVIEW_MODE'] != 'true'
```

Comment:
since this is a boolean value, I would prefer to state it as false instead of not true. What do you think?

Reply:
I’m convinced we don’t actually need it since we can just set the environment to preview or something